source/project/components/all-tasks/all-tasks.css

```diff
@@ -35,21 +35,27 @@ body,
 	justify-content: space-between;
 	padding: 10px 20px;
 	border-bottom: 1px solid var(--border-color);
+	height: 78px;
+}
+
+.logo {
+	margin-left: 5px;
```

Comment:
Interesting that we're using absolute instead of relative, hope this stays the same at diff screen sizes

Reply:
oh shoot should i change that

Reply:
Should be chill.

Reply:
Just checked actually and it stays